lib/internal/NativeWrapper.d.ts

```diff
@@ -0,0 +1,42 @@
+import { PowerAuthError } from '../model/PowerAuthError';
```

Comment:
I would consider removing this file. I did it manually in the last release to hide the API from the user as it's internal. It has basically the same concept as private headers/APIs on iOS.